xdsl/transforms/experimental/ConvertStencilToLLMLIR.py

```diff
@@ -0,0 +1,68 @@
+from typing import TypeVar
+
+from xdsl.pattern_rewriter import (PatternRewriter, PatternRewriteWalker,
+                                   RewritePattern, GreedyRewritePatternApplier)
+from xdsl.ir import MLContext, Operation
+from xdsl.irdl import Attribute
+from xdsl.dialects.builtin import ArrayAttr, FunctionType, IntegerAttr, ModuleOp, AnyIntegerAttr, IndexType
+from xdsl.dialects.func import FuncOp
+from xdsl.dialects.memref import MemRefType
+
+from xdsl.dialects.experimental.stencil import FieldType, IndexAttr
+
+_TypeElement = TypeVar("_TypeElement", bound=Attribute)
+
+
+def GetMemRefFromField(
+        inputFieldType: FieldType[_TypeElement]) -> MemRefType[_TypeElement]:
+    memref_shape_integer_attr_list: list[AnyIntegerAttr] = []
+    for i in inputFieldType.shape.data:
+        memref_shape_integer_attr_list.append(
+            IntegerAttr.from_params(i.value.data, i.typ))
+
+    memref_shape_array_attr = ArrayAttr[AnyIntegerAttr](
+        memref_shape_integer_attr_list)
+
+    return MemRefType.from_params(inputFieldType.element_type,
+                                  memref_shape_array_attr)
+
+
+def GetMemRefFromFieldWithLBAndUB(memref_element_type: _TypeElement,
```

Comment:
Feels like these might be useful as accessors in memeref?

We cna also leave them here for now, they don't need tests here I guess...

Reply:
No, those IndexAttr are stencil.IndexAttr. I guess this is a point for always prefixing those with the dialect for readability though!